libs/wire-api/src/Wire/API/Conversation/Protocol.hs

```diff
@@ -144,3 +159,15 @@ deriving via (Schema Protocol) instance ToJSON Protocol
 protocolDataSchema :: ProtocolTag -> ObjectSchema SwaggerDoc Protocol
 protocolDataSchema ProtocolProteusTag = tag _ProtocolProteus (pure ())
 protocolDataSchema ProtocolMLSTag = tag _ProtocolMLS mlsDataSchema
+protocolDataSchema ProtocolMixedTag = tag _ProtocolMixed mlsDataSchema
+
+newtype ProtocolUpdate = ProtocolUpdate {unProtocolUpdate :: ProtocolTag}
```

Comment:
For future-proofing, maybe we should wrap the JSON instance in an object.

Reply:
That was my intention, even with the newtype wrapper. But I made a mistake in the schema definition. Fixed now